### PR TITLE
cpp api: add support for openvino

### PR DESCRIPTION
**Pull request by zzk0**

### CODE REVIEW

**Review by hjchen2 — APPROVED** [review]

File: oneflow/xrt/openvino/ops/batch_norm_op.cpp [hunk 1]
```diff
@@ -32,7 +32,7 @@ class NormalizationOp : public OpenvinoOpKernel {
     std::shared_ptr<ngraph::Node> moving_mean = ctx->Weight("moving_mean_0");
     std::shared_ptr<ngraph::Node> moving_variance = ctx->Weight("moving_variance_0");
     float epsilon = ctx->Attr<float>("epsilon");
-    std::shared_ptr<ngraph::Node> ngraph_node = std::make_shared<ngraph::op::BatchNormInference>(
+    std::shared_ptr<ngraph::Node> ngraph_node = std::make_shared<ngraph::op::v0::BatchNormInference>(
```
Comment (daquexian) [inline oneflow/xrt/openvino/ops/batch_norm_op.cpp:35]: 这里 v0 是什么版本，我看有 v0 v1 v3 v4 v5 v6 v7

**Review by mosout — APPROVED** [review]

File: oneflow/xrt/openvino/ops/batch_norm_op.cpp (hunk 1 above)
Comment (zzk0) [reply oneflow/xrt/openvino/ops/batch_norm_op.cpp:35]: 不同版本有个 opset，可能是通过增加新的命名空间来支持新的 opset

BatchNormInference 有可能是在项目启动的时候就已经有了，并且当时没有用这种 v0 v1... v7 的方式来扩展管理 opset，所以后面把它加到了 v0。（我猜

Comment (jackalcooper) [reply oneflow/xrt/openvino/ops/batch_norm_op.cpp:35]: > 不同版本有个 opset，可能是通过增加新的命名空间来支持新的 opset
> 
> BatchNormInference 有可能是在项目启动的时候就已经有了，并且当时没有用这种 v0 v1... v7 的方式来扩展管理 opset，所以后面把它加到了 v0。（我猜

全部都加v0就可以用2021.4吗

Comment (daquexian) [reply oneflow/xrt/openvino/ops/batch_norm_op.cpp:35]: 那如果用 v1 v3 等等可以吗

Comment (zzk0) [reply oneflow/xrt/openvino/ops/batch_norm_op.cpp:35]: 不可以，不同时期加入的算子会放在不同的命名空间。比如在 opset7 中新支持的算子，就加入到 v7 里面；像 BatchNormInference 可能是一开始就有的，不过那时候没有命名空间，现在加上了。

> 全部都加v0就可以用2021.4吗

现在这个已经可以用 2021.4 了（我也没想到原来只要加一个命名空间就好了
